aws/environments/stomlinson.yml

```diff
@@ -14,8 +14,8 @@ hosted_zone: lcip.org
 # ssl_certificate_arn: arn:aws:acm:us-west-2:927034868273:certificate/ef6bb380-b02e-4433-9355-aec09dedd381
 ssl_certificate_arn: arn:aws:acm:us-east-1:927034868273:certificate/675e0ac8-23af-4153-8295-acb28ccc9f0f
 
-content_git_version: master
-fxadev_git_version: docker
+content_git_version: email-first 
```

Comment:
this is not going to work, needs to be a `feature.` branch, so there is a docker tag